content/en/apps/payments/send-payments.md

```diff
@@ -0,0 +1,261 @@
+---
+date: '2020-01-08T09:59:25Z'
+menu:
+  apps:
+    parent: "payments"
+    name: Send payments locally
+title: Send and receive payments with the demo CorDapps
+weight: 300
+---
+
+You can use the Corda Payments Technical Preview to locally replicate payments in a live environment.   
+
+By following this guide, you can:
+
+1. Deploy a local network that includes the required nodes to sent and receive payments, and a node to act as the Payments Agent.
+2. Set up the Payments Agent locally.
+3. Add the details of two Modulr accounts to the payer and payee nodes on your local network.
+4. Make a payment by initiating a payment flow.
+5. Set triggers for call back listeners that can trigger flows at specific points in the payments process.
+
+## Requirements
+
+You must have :
+
+* Access to the Corda Enterprise Artifactory repository.
+* A Corda Payments-enabled CorDapp.
+* Modulr sandbox credentials.
+
+## About Modulr accounts and payment accounts
+
+Payments can only be triggered by a member of a network if they have an existing account with the PSP, and these details are reflected in the Payments Agent CorDapp.
+
+When using the Corda Payments API method `createPaymentAccount`, one required parameter is the `accountName` - this is the name for the account that is registered with Modulr.
```

Comment:
Tim is correct, it is called addPaymentAccount. 

This suggests that there is just 1 required parameter but there are several for a payment account

	private val accountName: String,
	private val currency: FiatCurrency,
	private val psp: String,
	private val pspAccountId: String,
	private val credentials: Map<String, String>